docs/conf.py

```diff
@@ -54,6 +54,8 @@
     "sphinx_copybutton",
     # Adds a copy button to each code block in the documentation
     # https://sphinx-copybutton.readthedocs.io/en/latest/
+    "sphinxcontrib.bibtex",
+    # Allows us to add references to a page
```

Comment:
The comment should be before the value:
```suggestion
    # Allows us to add references to a page
    "sphinxcontrib.bibtex",
```
If you could also fix that for the entry just above it would be perfect.
Also, could you include an URL to the `sphinxcontrib.bibtex` documentation?